cmd/kubeadm/app/util/config/initconfiguration.go

```diff
@@ -142,7 +142,7 @@ func SetAPIEndpointDynamicDefaults(cfg *kubeadmapi.APIEndpoint) error {
 
 	// kubeadm allows users to specify address=Loopback as a selector for global unicast IP address that can be found on loopback interface.
 	// e.g. This is required for network setups where default routes are present, but network interfaces use only link-local addresses (e.g. as described in RFC5549).
-	if addressIP.IsLoopback() {
+	if addressIP != nil && addressIP.IsLoopback() {
```

Comment:
has this been panicing under some conditions?

please include details in the PR description, perhaps also reproduction steps (given there is no linked issue).

Reply:
No use case but I just found a potential bug here

Reply:
Line 139 already validates this potential bug

Reply:
yes, for "" as input, nil will always be returned.
@PascalBourdier please send other fixes, this is not needed.
/close